man/docker-build.1.md

```diff
@@ -235,6 +235,12 @@ If you do not provide a version tag then Docker will assign `latest`:
 
 When you list the images, the image above will have the tag `latest`.
 
+Sometimes you may want to update the `latest` tag to point to the
+newly built image as well as keeping a long term tag to reference a specific
+version. This can be done by :
+
+    docker build -t whenry/fedora-jboss:v2.1 -t whenry/fedora-jboss:latest .
+
```

Comment:
@thaJeztah Please help to check if this sentence is correct. I'm not quite sure about the grammar :sweat: 
Thanks :smile: